ReactiveUI/AddTo.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace System.Reactive.Disposables
+{
+    public static class CompositeDisposableExtensions
+    {
+        /// <summary>
+        /// Adds a disposable to the specified <see cref="CompositeDisposable"/>.
+        /// </summary>
+        /// <typeparam name="T">
+        /// The type of the disposable.
+        /// </typeparam>
+        /// <param name="this">
+        /// The disposable.
+        /// </param>
+        /// <param name="compositeDisposable">
+        /// The <see cref="CompositeDisposable"/> to which <paramref name="this"/> will be added.
+        /// </param>
+        /// <returns>
+        /// The disposable.
+        /// </returns>
+        public static T AddTo<T>(this T @this, CompositeDisposable compositeDisposable)
```

Comment:
The original value is the disposable (this), just not the composite one, my remark is that the original value is unlikely to be of any use once it's put into the composite, hence not returning anything would make the user write the most sensible thing after calling that one, which is `;` :smile: